student.rb

```diff
@@ -8,4 +8,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true, id: ni
   def play_hooky
     'H¯\(ツ)/¯'
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.student.push(self) unless classroom.student.include?(self)
+  end
```

Comment:
- [ ] Kindly make sure that add `attr_reader :classroom` before initialize method. This is required for setting the classroom for students.